packages/dds/map/src/map.ts

```diff
@@ -312,6 +321,9 @@ export class SharedMap extends SharedObject<ISharedMapEvents> implements IShared
         };
         builder.addBlob(snapshotFileName, JSON.stringify(header));
 
+        totalBlobBytes += currentSize;
```

Comment:
Same feedback as in Sequence - it would be great for SharedObject (who calls this method) to do this kind of work for all DDSs who use it as a base class, and not need to add this code everywhere.